manifests/s/Sass/Dart-Sass/1.83.4/Sass.Dart-Sass.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Sass.Dart-Sass
+PackageVersion: 1.83.4
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: dart-sass\src\dart.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/sass/dart-sass/releases/download/1.83.4/dart-sass-1.83.4-windows-x64.zip
+  InstallerSha256: 89FC2218E137C67229A0FC163EEAC44EFA1815139CBA9080208935DEBD55F732
+- Architecture: x86
+  InstallerUrl: https://github.com/sass/dart-sass/releases/download/1.83.4/dart-sass-1.83.4-windows-ia32.zip
+  InstallerSha256: A74E939E7F804EC8E364C3B5EFE4E66BA172BECF4D69B36511EC60017573218F
+- Architecture: arm64
+  InstallerUrl: https://github.com/sass/dart-sass/releases/download/1.83.4/dart-sass-1.83.4-windows-arm64.zip
+  InstallerSha256: 7B35198A7903A543F157E4C572374E29AF5F0E995A8D291EB114B42310E00D7D
+ManifestType: installer
+ManifestVersion: 1.9.0
+ReleaseDate: 2025-01-14

```

```suggestion
ReleaseDate: 2025-01-14
UpgradeBehavior: uninstallPrevious
```

I think this is intended. If we leave this un-configured or set it to `install`, it might cause unwanted side-by-side installation problem.